manual_tests/internal/create_data_action.go

```diff
@@ -0,0 +1,59 @@
+package internal
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/bsv-blockchain/go-sdk/transaction"
+	sdk "github.com/bsv-blockchain/go-sdk/wallet"
+	"github.com/bsv-blockchain/go-wallet-toolbox-manual-tests/internal/fixtures"
+	"github.com/go-softwarelab/common/pkg/to"
+)
+
+func (m *Manager) CreateActionWithData(user fixtures.UserConfig, data string) (string, fixtures.Summary, error) {
+	var summary fixtures.Summary
+
+	summary = append(summary, fmt.Sprintf("Using wallet for user %s", user.Name))
+
+	userWallet, err := m.WalletForUser(user)
+	if err != nil {
+		return "", summary, fmt.Errorf("failed to get wallet for user %s: %w", user.Name, err)
+	}
+
+	summary = append(summary, fmt.Sprintf("Creating data output with data: %s", data))
+
+	dataOutput, err := transaction.CreateOpReturnOutput([][]byte{[]byte(data)})
+	if err != nil {
+		return "", summary, fmt.Errorf("failed to create OP_RETURN output with data %q: %w", data, err)
+	}
+
```

Comment:
consider log the data output contents directly like below ? 
```suggestion
summary = append(summary, fmt.Sprintf("OP_RETURN script: %x", dataOutput.LockingScript.Bytes()))
```